test/framework/src/main/java/org/elasticsearch/index/replication/ESIndexLevelReplicationTestCase.java

```diff
@@ -197,7 +197,7 @@ public void backgroundSync(ShardId shardId, RetentionLeases retentionLeases) {
             }
         };
 
-        protected ReplicationGroup(final IndexMetaData indexMetaData) throws IOException {
+        public ReplicationGroup(final IndexMetaData indexMetaData) throws IOException {
```

Comment:
is this change necessary? I thought it was accessed from a subclass?

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/44989/commits/1b91900e403e16acefad70d50ab5009a5a690f3f.